### PR TITLE
split up circleCI jobs into parallel workflows

### PR DESCRIPTION
**Pull request by Dieterbe**
continuation of #803

### CODE REVIEW

**Comment by Dieterbe** [comment]
this workflow stuff in circleCI is neat.
click on the green checkmark to see individual workflows, from there going to the workflow overview is a bit trickier, but here's the link for your convenience : https://circleci.com/workflow-run/a9e3cbc0-2925-4a9c-9c28-594faa778218

**Comment by Dieterbe** [comment]
i still need to make sure the tags are re-instated properly, will circle back (no pun intented) after i'm done with some other pr's

**Comment by Dieterbe** [comment]
@woodsaj mergey mergey?

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,12 +39,25 @@ jobs:
...
+      - run: docker load -i build_docker/metrictank.tar
       - run: scripts/qa/end2end.sh
+
+  deploy:
+    docker:
+      - image: circleci/ruby:2.3
+    steps:
+      - checkout
+      - attach_workspace:
+          at: build
```
Comment (woodsaj) [inline .circleci/config.yml]: I think this should be `at: .`  otherwise you will end up with `build/builld/metrictank`

**Pull request merged**